atm.js

```diff
@@ -1,21 +1,99 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
 
-var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
 
-}
+// ======= init ======= 
+var Cbalance = parseInt($('.balance').eq(0).html().substring(1));
+var Sbalance = parseInt($('.balance').eq(1).html().substring(1));
 
-function checkingWithdrawal(amount){
 
-}
+// ======= buttons ======= 
+var Cdeposit = $('.account input').eq(1).on('click', function(event) {
+	make_transaction('checking', 'deposit');
+});
 
-function savingsDeposit(amount){
+var Sdeposit = $('.account input').eq(4).on('click', function(event) {
+	make_transaction('savings', 'deposit')
+});
 
-}
+var Cwithdraw = $('.account input').eq(2).on('click', function(event){
+	make_transaction('checking', 'withdraw');
+});
+
+var Swithdraw = $('.account input').eq(5).on('click', function(event){
+	make_transaction('savings', 'withdraw');
```

Comment:
I really like how for each event listener, you just call one function, this really keeps your UI code nice and short/tidy.